libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -369,6 +369,26 @@ bool AP_Arming::gps_checks(bool report)
             return false;
         }
     }
+
+    // check GPSs are within 50m of each other and that blending is healthy
+    if ((checks_to_perform & ARMING_CHECK_ALL) || (checks_to_perform & ARMING_CHECK_GPS_CONFIG)) {
+        float distance_m;
+        if (!gps.all_consistent(distance_m)) {
+            if (report) {
+                GCS_MAVLINK::send_statustext_all(MAV_SEVERITY_CRITICAL,
+                                                 "PreArm: GPSs inconsistent by %4.1fm",
```

Comment:
Thanks! I admit I only ever plan these messages to be relative to the statustext length. MP could break into 2 lines anything that is to long to just fit in the HUD, but thats a  separate thing, and has its own complications with doing that.